addons/hr/views/hr_employee_views.xml

```diff
@@ -352,6 +352,31 @@
             </field>
         </record>
 
+        <!-- This part of the view_employee_form is defined separately so that the
+        smartbutton can have lower priority and therefore be last in the list. -->
+        <record id="view_employee_form_smartbutton_inherited" model="ir.ui.view">
```

Comment:
Why are you using inheritance? The main employee view is there, you could directly change the original one

Reply:
We don't want to show the contact button in the main menu frame
It would be better to show it in the 'more' menu list

Thank you :innocent: